README.md

```diff
@@ -44,18 +44,39 @@ pip install -i https://test.pypi.org/simple/ pyimager
 
 
 ## Dependencies:
-- Python 3.6 or greater
-    - numpy
-    - PIL
-    - scipy
-    - matplotlib
+- Python 3.7 or greater
+    - numpy = "^1.18.1"
+    - Pillow = "^7.0.0"
+    - scipy = "^1.4.1"
+    - matplotlib = "^3.2.0"
 
-## Usage:
+## Usage Examples 
 
-``` python
+```python
 from pyimager import pyimager
+
+pyimager.circropper("images/bear.jpg", 0)
+```
+![](images/bear.jpg) 
```

Comment:
```suggestion
![](images/mandrill.jpg) 
```

Reply:
I have added two images in README, both bear, and you just give me one suggestion on changing the bear to mandrill, which will cause an unmatched result in this example. Would you mind changing this suggestion to "delete this image", and add one more suggestion on "delete the image after this one, 'images/result2.jpg'"? Thank you.